code_style_guideline/code_style_guideline.md

```diff
@@ -353,6 +385,59 @@ func performTask() -> Void {
 }
 ```
 
+Parameter Naming Convention:
+
+Mostly don't use ommited parameters on function. If parameter name contains on method name at it is clear to recognize. You can use.
+ 
+ **Preferred**:
+```swift
+func addImage(_ image: UIImage) {
+    ...
+}
+
+func addImage(with image: UIImage) {
```

Comment:
I think we need to discuss this example, imo, with is not redundant here but it decreases readability